README.md

```diff
@@ -23,10 +22,20 @@ Example Playbook
 
     - hosts: servers
       roles:
-         -
-           role: devops_cmp.ansible_filebeat
-           version: 6.5.4
-           log_tags: ['prod', 'myapp', 'mycomponent']
+         - role: devops_cmp.ansible_filebeat
+           version: 7.5.0
+           processors:
+             - paths: ['/var/log/httpd/*access.log']
+               tags: ['httpd', 'access', 'isprime', '2.2']
```

Comment:
remove isprime to not give extra info on the role about us

Reply:
done